### DISCUSSION TITLE
Calculating percentiles and mean

### DISCUSSION THREAD

**Discussion by kishaningithub** [saulpw/visidata#1062] [Q&A]
I have a list of latency data (basically a number)

I would like to know the following

1. Average/Mean
2. 95th Percentile
3. 99th percentile

Is this possible?

**Comment by frosencrantz** [comment]
Hi @kishaningithub !  There is support for that.  

Take a look at aggregator support: https://visidata.org/docs/group/

Using the "+" shortcut you can add aggregators when using the frequency table.
And the "z+" shortcut to get it for all selected rows.

For the fine-grained percentiles see Saul's answer here: https://github.com/saulpw/visidata/issues/571
